jazz_asset-event-handler/components/config.js

```diff
@@ -0,0 +1,38 @@
+// =========================================================================
+// Copyright © 2017 T-Mobile USA, Inc.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// =========================================================================
+
+var getStageConfig = (context) => {
```

Comment:
Lets follow one pattern - https://github.com/tmobile/jazz/blob/master/jazz_deployments/components/config.js

Reply:
I have looked at the above config.  Why we add AWS specific things in a generic reusable component?  For example:  if (event && event.awslogs && event.awslogs.data) {

Reply:
Cleaned up the config.  As discussed we cant use the above file as it is.

Reply:
We want to consolidate and have a single version of this file as opposed to different variants. 

`event` is a json object and reading a property called awslogs doesn't make it AWS specific (unless you are thinking in Java). With that line of thought `context.functionName` also makes it specific to aws. The idea of this common lib should be it works for the consumer w/o they having to know about the specifics. 

Lets apply the change in this PR that makes it work for all "dev" environments on top of this file: https://github.com/tmobile/jazz/blob/master/jazz_deployments/components/config.js.

Reply:
I think this is not consistent across board?  Example: https://github.com/tmobile/jazz/blob/v1.7.1/jazz_services/components/config.js. How this works without that hack ?

I need to understand better before I make this change.  What is meant by if event has 'awslogs.data' in it makes it run with DEV properties 

if (event && event.awslogs && event.awslogs.data) {
    // cw events default to dev
    stage = 'dev';

Reply:
The newer services have a better version of the file (services is one of the earliest microservices hence you don't see it there) and the recommendation in this thread was to use that. What I see here though is that we have created a new version of the file instead of building on top of the earlier versions. 

From the latest comment I gather that there was a need for understanding why and how this works. In that case did you seek clarification and/or tried to understand that before creating a newer variation and merging the code in?